### PR TITLE
SLCORE-708 Update apache commons-compress

### PR DESCRIPTION
**Pull request by vojtech-suchy-sonarsource**

### CODE REVIEW

**Review by thahnen — APPROVED** [review]
LGTM!

File: client/java-client-osgi/shared.bnd [hunk 1]
```diff
@@ -5,3 +5,4 @@ Bundle-Version: ${parsedVersion.osgiVersion}
 # BND configuration to tweak generation of Manifest entries for the normal / sources JAR archive
 -removeheaders: Bnd-LastModified,Bundle-Developers,Bundle-DocURL,Bundle-SCM,Include-Resource,Private-Package
 -noimportjava: true
+-fixupmessages "Classes found in the wrong directory"; restrict:=error; is:=warning
```
Comment (damien-urruty-sonarsource) [inline client/java-client-osgi/shared.bnd:8]: Why is this line needed in this context? Could we add a comment maybe?

**Review by damien-urruty-sonarsource — APPROVED** [review]

File: client/java-client-osgi/shared.bnd (hunk 1 above)
Comment (vojtech-suchy-sonarsource) [reply client/java-client-osgi/shared.bnd:9]: Added in [2a64f42](https://github.com/SonarSource/sonarlint-core/pull/973/commits/2a64f423d36f5f55e1ca9c31ae6e8f15ecb476f2)
